firebase-perf/src/test/java/com/google/firebase/perf/transport/RateLimiterTest.java

```diff
@@ -398,6 +398,67 @@ public void testDeviceSampling_tracesDisabledButNetworkEnabled() {
     assertThat(limiter.getIsDeviceAllowedToSendTraces()).isFalse();
   }
 
+  @Test
+  public void testDeviceSampling_tracesEnabledButFragmentDisabled() {
```

Comment:
nit: change the name to `testDeviceSampling_tracesEnabledButFragmentDisabled_dropsFragmentTrace`. A good name describes both the actions that are being taken on a system and the expected outcome.

go/unit-testing-practices

Reply:
Done